File: android/EpubRnD/src/com/hurix/epubRnD/VOs/HighlightVO.java
package com.hurix.epubRnD.VOs;

public class HighlightVO 
{
	private int _chapterIndex;
	private int _noOfPagesInChapter;
	private int _startWordID;
	private int _endWordID;
	private String _chapterPath;
	private String _selectedText;
	private boolean _hasNote;
	
	public int getChapterIndex() {
		return _chapterIndex;
	}
	public void setChapterIndex(int _chapterIndex) {
		this._chapterIndex = _chapterIndex;
	}
	public int getNumOfPagesInChapter() {
		return _noOfPagesInChapter;
	}
	public void setNumOfPagesInChapter(int _noOfPagesInChapter) {
		this._noOfPagesInChapter = _noOfPagesInChapter;
	}
	public int getStartWordID() {
		return _startWordID;
	}
	public void setStartWordID(int _startWordID) {
		this._startWordID = _startWordID;
	}
	public int getEndWordID() {
		return _endWordID;
	}
	public void setEndWordID(int _endWordID) {
		this._endWordID = _endWordID;
	}
	public String getChapterPath() {
		return _chapterPath;
	}
	public void setChapterPath(String _chapterPath) {
		this._chapterPath = _chapterPath;
	}
	public String getSelectedText() {
		return _selectedText;
	}
	public void setSelectedText(String _selectedText) {
		this._selectedText = _selectedText;
	}
	public boolean hasNote() {
		return _hasNote;
	}
	public void setHasNote(boolean _hasNote) {
		this._hasNote = _hasNote;
	}
	

}

File: ios/EpubRnD/EpubRnD/HighlightVO.m
//
//  HighlightVO.m
//  EpubRnD
//
//  Created by UdaySravan K on 17/07/14.
//  Copyright (c) 2014 hurix. All rights reserved.
//

#import "HighlightVO.h"

@implementation HighlightVO

- (void) setChapterIndex:(int) indexOfChapter
{
    _chapterIndex = indexOfChapter;
}

- (int) getChapterIndex
{
    return _chapterIndex;
}

- (void) setNoOfPagesInChapter:(int) noOfPagesInChapter
{
    _noOfPagesInChapter = noOfPagesInChapter;
}
- (int) getNoOfPagesInChapter
{
    return _noOfPagesInChapter;
}

- (void) setStartWordID:(int) startWordID
{
    _startWordID = startWordID;
}
- (int) getStartWordID
{
    return _startWordID;
}

- (void) setEndWordID:(int) endWordID
{
    _endWordID = endWordID;
}
- (int) getEndWordID
{
    return _endWordID;
}
    
-(void)setHasNote:(BOOL)hasNote
{
    _hasNote = hasNote;
}
    
-(BOOL)hasNote
{
    return _hasNote;
}

@end
